examples/cryptocurrency/tests/api.rs

```diff
@@ -227,28 +228,29 @@ impl CryptocurrencyApi {
     /// within the response).
     /// Note that the transaction is not immediately added to the blockchain, but rather is put
     /// to the pool of unconfirmed transactions.
-    fn create_wallet(&self, name: &str) -> (TxCreateWallet, SecretKey) {
+    fn create_wallet(&self, name: &str) -> (Message<RawTransaction>, SecretKey) {
         let (pubkey, key) = crypto::gen_keypair();
         // Create a pre-signed transaction
-        let tx = TxCreateWallet::new(&pubkey, name, &key);
-
+        let tx = TxCreateWallet::sign(name, &pubkey, &key);
+        let data = hex::encode(tx.clone().serialize());
```

Comment:
I remember we have some utilities and traits related to hexadecimal conversions in `exonum::encoding`. For one thing, I think we even reexport `hex::encode` as `exonum::encoding::serialize::encode_hex`.

Reply:
Yeap we can, but encoding is module of our serialisation, and i am not sure we should reexport items that does not belong to this concrete serialisation.

What do you think? @DarkEld3r @ilammy

Reply:
Well... but we _do_ send these transactions to Exonum. Therefore it is expected that transactions should be encoded in some Exonum-specific way (lowercase base16 in this case). So it would not be unusual for Exonum to provide some readymade tools for encoding.

I don't have any strong opinion on this matter. I'm totally fine with using `hex` crate for base16 encoding if the web API requires the request to be JSONs of the form `{ "tx_body": "7472616e73616374696f6e20626f647920627974657320617320612068657861646563696d616c20737472696e6720616b6120626173653136" }` Any library or handwritten function would do fine, I guess. I just wondered whether we already have something specifically meant for this use case. Because [messages seem to use `hex` internally](https://github.com/exonum/exonum/blob/e98cb28ce21e3237d1b63f45c535a342a1b20a50/exonum/src/messages/mod.rs#L218-L237) it may be a good fit.